homeassistant/components/transmission/switch.py

```diff
@@ -40,30 +40,20 @@ class TransmissionSwitch(SwitchEntity):
 
     def __init__(self, switch_type, switch_name, tm_client, client_name):
         """Initialize the Transmission switch."""
-        self._name = switch_name
-        self.client_name = client_name
+        self._attr_name = f"{client_name} {switch_name}"
```

Comment:
Please don't add this unwanted composing of name with device name.
Instead, leave it as is and in another PR set `_attr_has_entity_name = True` to have the name composing happen automatically in frontend.